docs/_index.md

```diff
@@ -22,48 +22,48 @@ outputs:
 <div class="cards max-page use-cases">
   <div class="row">
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/configure/" class="noanchor"><div>
+  <a href="/get-started/try-viam/" class="noanchor"><div>
     <picture><img src="icons/rover.svg" alt="Rover" loading="lazy"></picture>
     <p>Try Viam</p></div>
   </a>
 </div>
 <div class="col hover-card link-with-icon">
-  <a href="use-cases/configure/" class="noanchor"><div>
+  <a href="/use-cases/configure/" class="noanchor"><div>
```

Comment:
why did you add all the leading slashes? They do mean things on staging break unfortunately, which is why we don't use them in html

Reply:
oh ok, didn't know that

Reply:
fixed!